agent/tests/test_php_execute.c

```diff
@@ -287,7 +269,7 @@ static void test_stack_depth_after_exception() {
   arg = tlib_php_request_eval_expr("0");
   expr = nr_php_call(NULL, "followup_uncaught", arg);
   tlib_pass_if_int_equal(
-      "PHP stack depth tracking should be 2 after function call", 2,
+      "PHP stack depth tracking should be 2 after function call", 0,
```

Comment:
Should this unit test description string be updated to `0` as well?
```suggestion
      "PHP stack depth tracking should be 0 after function call", 0,
```